tests/unit/no_cheat.py

```diff
@@ -14,9 +18,11 @@ def no_cheat(diff_text: str) -> str:
         idx = line.find(DISABLE_TAG)
         if idx < 0:
             continue
-        codes = line[idx + len(DISABLE_TAG) :].split(',')
+        codes = {_strip_code(code) for code in line[idx + len(DISABLE_TAG) :].split(',')}
+        allowed_local_cyclic_imports = {'cyclic-import', 'import-outside-toplevel'}
+        if len(codes.intersection(allowed_local_cyclic_imports)) == 2:
```

Comment:
```suggestion
        if len(codes.intersection(allowed_local_cyclic_imports)) == len(allowed_local_cyclic_imports):
```
to make future changes easier